src/main/java/namor/template/calculator/ClassicCalculator.java

```diff
@@ -0,0 +1,87 @@
+package namor.template.calculator;
+
+import namor.template.stack.Stack;
+
+public class ClassicCalculator {
+
+    private Stack<Double> operands;
```

Comment:
Stack is not the only collection you can use in your code. Maybe you should rethink and find collection that fits better in this task.

Reply:
But it still have to implement Stack in runtime?